GameServer/Logic/Config.cs

```diff
@@ -128,12 +128,34 @@ public class PlayerConfig
         public string Summoner2 { get { return (string)_playerData.SelectToken("summoner2"); } }
         public short Ribbon { get { return (short)_playerData.SelectToken("ribbon"); } }
         public int Icon { get { return (int)_playerData.SelectToken("icon"); } }
+        public Dictionary<string, List<int>> Runes { get { return _runeList; } }
 
         private JToken _playerData;
+        private Dictionary<string, List<int>> _runeList;
 
         public PlayerConfig(JToken playerData)
         {
-            _playerData = playerData;
+            _playerData = playerData;
+            try
+            {
+                var runes = _playerData.SelectToken("runes");
+                _runeList = new Dictionary<string, List<int>>();
+
+                foreach (JProperty runeCategory in runes)

```

Comment:
Also if you do create the `RuneCollection` class, you should move the loading code probably there. You can actually do the whole loading with a single JSON.net function if your data structure and object structure match.